### ISSUE TITLE
Regression: False positive I1101(c-extension-no-member) for np.finfo(np.float64).resolution

### ISSUE THREAD

**Issue by sam-s** [PyCQA/pylint#5831]
### Bug description

""
import numpy as np
print(np.finfo(np.float64).resolution)

### Configuration

```ini
extension-pkg-allow-list=numpy
```

### Command used

```shell
pylint z.py
```

### Pylint output

```shell
************* Module z
z.py:3: [E1101(no-member), ] Instance of 'finfo' has no 'resolution' member
```

### Expected behavior

no diagnostics

### Pylint version

```shell
pylint 2.12.2
astroid 2.9.3
Python 3.9.10 (main, Jan 15 2022, 11:48:00) 
[Clang 13.0.0 (clang-1300.0.29.3)]
```

### OS / Environment

mac os

### Additional dependencies

numpy==1.22.2

**Comment by sam-s** [comment]
Also reported in https://github.com/PyCQA/pylint/issues/5081#issuecomment-928289526

**Comment by dpinol** [comment]
Same for min & max members
